cpp/test/sg/tsne_test.cu

```diff
@@ -123,8 +125,8 @@ class TSNETest : public ::testing::Test {
 
 typedef TSNETest TSNETestF;
 TEST_F(TSNETestF, Result) {
-  if (score_bh < 0.98) printf("BH score = %f\n", score_bh);
-  if (score_exact < 0.98) printf("Exact score = %f\n", score_exact);
+  if (score_bh < 0.98) CUML_LOG_INFO("BH score = %f", score_bh);
```

Comment:
Same here- IMO, this should be debug as well.

Reply:
Done.